INDEX.md

```diff
@@ -0,0 +1,78 @@
+grow your Flutter skills. Some are maintained here by the Flutter team,
+but many have been created by the Flutter community and are kept in other repos
+in and out of GitHub.
+
+This is not an exhaustive list of samples, and just because because a project
+isn't listed here doesn't mean that it's not worth exploring. Similarly, while
+the Flutter team works to keep this list up to date, there are plenty of others
+created by the community, such as
+[Awesome Flutter](https://github.com/Solido/awesome-flutter) from @Solido.
+
+## How-to Collections
+
+#### [Flutter Examples](https://github.com/nisrulz/flutter-examples) [![GitHub stars](https://img.shields.io/github/stars/nisrulz/flutter-examples.svg?style=social&label=Star)](https://github.com/nisrulz/flutter-examples) [![GitHub forks](https://img.shields.io/github/forks/nisrulz/flutter-examples.svg?style=social&label=Fork)](https://github.com/nisrulz/flutter-examples/fork)
+
+A collection of single-topic examples from Nishant Srivastava. These everything
+from gradients to JSON to routing and more.
+
+#### [Flutter Example Apps](https://github.com/iampawan/FlutterExampleApps) [![GitHub stars](https://img.shields.io/github/stars/iampawan/FlutterExampleApps.svg?style=social&label=Star)](https://github.com/iampawan/FlutterExampleApps) [![GitHub forks](https://img.shields.io/github/forks/iampawan/FlutterExampleApps.svg?style=social&label=Fork)](https://github.com/iampawan/FlutterExampleApps/fork)
+
+Several dozen examples from Flutter GDE Pawan Kumar. These cover state
+management, Firebase, UI design, and many other topics. Each one comes with a
+YouTube videos showing implementation.
+
+#### [Flutter by Example](https://github.com/mjohnsullivan/flutter-by-example) [![GitHub stars](https://img.shields.io/github/stars/mjohnsullivan/flutter-by-example.svg?style=social&label=Star)](https://github.com/mjohnsullivan/flutter-by-example) [![GitHub forks](https://img.shields.io/github/forks/mjohnsullivan/flutter-by-example.svg?style=social&label=Fork)](https://github.com/mjohnsullivan/flutter-by-example/fork)
+
+Twenty-odd samples detailing common tasks with Flutter: how to use text fields,
+streams and StreamBuilders, and more.
+
+## Architecture / networking / backend
+
+#### [Flutter Architectural Samples](https://github.com/brianegan/flutter_architecture_samples) [![GitHub stars](https://img.shields.io/github/stars/brianegan/flutter_architecture_samples.svg?style=social&label=Star)](https://github.com/brianegan/flutter_architecture_samples) [![GitHub forks](https://img.shields.io/github/forks/brianegan/flutter_architecture_samples.svg?style=social&label=Fork)](https://github.com/brianegan/flutter_architecture_samples/fork)
+
+Brian Egan's implementations of TodoMVC using a variety of state management and
+architectural approaches. If you'd like to see how an app built with
+`flutter-redux` is different than one using BLoC, this is a great place to
```

Comment:
is different from - https://www.grammar.com/different-from-vs-different-than

Reply:
You can really tell I was an English major, can't you? :)